modules/apps/frontend-js/frontend-js-test/src/testFunctional/macros/Walkthrough.macro

```diff
@@ -1,5 +1,16 @@
 definition {
 
+	macro assertEnablePopoverModeNotPresent {
+		if (IsElementNotPresent(locator1 = "Walkthrough#POPOVER")) {
+			if (IsElementNotPresent(key_portletName = "JS Walkthrough Sample", locator1 = "Portlet#TITLE")) {
+				IsElementNotPresent(locator1 = "Walkthrough#HOTSPOT");
+			}
+			else {
+				IsElementNotPresent(locator1 = "Walkthrough#SAMPLE_PORTLET_HOTSPOT");
+			}
+		}
+	}
+
```

Comment:
I think the if else statements are not needed. And does not need to be a macro. 

Also, there's no assertions in the macro they're all IsElementNotPresent logic checks.